swan-cern/scripts/before-notebook.d/11_htcondor.sh

```diff
@@ -17,6 +17,9 @@ then
   ln -s /eos/project/l/lxbatch/public/config-condor-swan/ngbauth-submit /etc/sysconfig/ngbauth-submit
 
   # Create self-signed certificate for Dask processes
+  export DASK_TLS_DIR=/srv/dask/tls # export so SwanHTCondorCluster finds the certs
```

Comment:
Why can't this no longer be created in the image? This should be super fast to run, but it would be one less thing at runtime.

Reply:
As discussed, because it needs to be chowned to the right $NB_USER:$NB_GID.